sdk/aqueduct/utils.py

```diff
@@ -121,8 +121,11 @@ def schedule_from_cron_string(schedule_str: str) -> Schedule:
     return Schedule(trigger=TriggerType.PERIODIC, cron_schedule=schedule_str)
 
 
-def retention_policy_from_latest_runs(k_latest_runs: int) -> RetentionPolicy:
-    return RetentionPolicy(k_latest_runs=k_latest_runs)
+def retention_policy_from_latest_runs(config: Optional[FlowConfig]) -> RetentionPolicy:
```

Comment:
Let's try and keep the parameters for this function the same and move the logic in determining whether the `config`/`config.k_latest_runs` exist in `publish_flow`.